src/routes/ProfessionSelectionScreen.tsx

```diff
@@ -47,10 +47,10 @@ const ProfessionSelectionScreen = ({ route, navigation }: ProfessionSelectionScr
   const isSelectionMade = selectedProfessions && selectedProfessions.length > 0
 
   const selectDiscipline = async (selectedItem: Discipline): Promise<void> => {
-    if (selectedProfessions?.some(profession => profession.id === selectedItem.id)) {
-      await AsyncStorage.removeSelectedProfession(selectedItem).then(refreshSelectedProfessions)
+    if (selectedProfessions?.includes(selectedItem.id)) {
```

Comment:
You cannot click it. Behaviour should be:
For non-initial view:
- if a profession is selected, it can be unselected by clicking the x-button on the ManageSelectionScreen
- if in the ProfessionSelectionScreen only more elements can be selected, but not unselected, therefore the already selected items are disabled and therefor cannot be unselected
For initial view:
- it is possible to select and unselect items, therefore we need the check in this function. An item is only disabled for non initial views.